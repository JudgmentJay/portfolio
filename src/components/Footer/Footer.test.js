import React from 'react'
import renderer from 'react-test-renderer'

import { Footer } from '../'

const now = new Date()

it('Renders correctly', () => {
	const tree = renderer
		.create(<Footer date={now} />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
