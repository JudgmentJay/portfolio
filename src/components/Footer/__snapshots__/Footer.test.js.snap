// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
[
  <footer
    className="footer"
  >
    ©2018–
    2023
     Jay Posten
  </footer>,
  <script
    src={{}}
    type="text/javascript"
  />,
]
`;
